public/app/features/dimensions/editors/FileUploader.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { FileDropzone, useTheme2, Button, DropzoneFile } from '@grafana/ui';
+import { getBackendSrv, config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { MediaType } from '../types';
+interface Props {
+  setNewValue: Dispatch<SetStateAction<string>>;
+  setFormData: Dispatch<SetStateAction<FormData>>;
+  mediaType: MediaType;
+  setUpload: Dispatch<SetStateAction<boolean>>;
+  getRequest: (formData: FormData) => Promise<UploadResponse>;
+}
+interface UploadResponse {
+  err: boolean;
+  path: string;
+}
+export function FileDropzoneCustomChildren({ secondaryText = 'Drag and drop here or browse' }) {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
```

Comment:
```suggestion
  const styles = useStyles2(getStyles);
```
We have a custom hook that makes this a bit easier.